### ISSUE TITLE
using EMGUCV and MAUI razor

### ISSUE THREAD

**Issue by onimor** [emgucv/emgucv#754]
How can I show video from the camera using EMGUCV and MAUI razor ?
the mat to base64 converter spends a lot of resources and has problems with speed.
Do you already have some way or ideas?

**Comment by emgucv** [comment]
We do not officially support MAUI blazer / razor at this moment.

**Comment by onimor** [comment]
> We do not officially support MAUI blazer / razor at this moment.

Are there any plans?

**Comment by YoungRicky7** [comment]
Does it support MAUI?

**Comment by emgucv** [comment]
Yes. MAUI is supported. You can try it out by adding the maui mini runtime package from nuget.org:
https://www.nuget.org/packages/Emgu.CV.runtime.maui.mini

The mini runtime let you use the opencv core modules in MAUI. The commercial license will let you use all modules.

**Comment by emgucv** [comment]
MAUI is officially supported. You can get started with the NuGet packages on nuget.org:

- [`Emgu.CV.runtime.maui.mini`](https://www.nuget.org/packages/Emgu.CV.runtime.maui.mini) — core OpenCV modules, free
- [`Emgu.CV.runtime.maui`](https://www.nuget.org/packages/Emgu.CV.runtime.maui) — full module set, requires a commercial license

For a working camera and image processing example, see the MAUI demo app in the repository under `Emgu.CV.Example/MAUI/MauiDemoApp/`.

Closing.
